merkletree/version.go

```diff
@@ -1,5 +1,7 @@
 package merkletree
 
 const (
-	Version = "0.1"
+	// Version indicates the current version of the library.
+	// This is matched with our release version on Github.
+	Version = "0.1.0"
```

Comment:
Hmm, this is included in the STR.  Should that really be the library version or a protocol version?